Populate dropdowns from tables in the search dataset

It would be best to only ever query the `search` dataset from this app, so that it's easier to reason about the data pipeline.

https://github.com/alphagov/govuk-knowledge-graph-search/blob/17feed8abdc3b6bb18d8e0838a93657d6e1cb817/bigquery.ts#L80-L91

```[tasklist]
### Tasks
- [ ] locale
- [x] taxon
- [ ] organisation
```

Cypress test fails for the infobox for taxons.

```
  Infobox results
    ✓ Shows the Person infobox (2043ms)
    ✓ Shows the Role infobox (2037ms)
    1) Shows the Taxon infobox
    ✓ Shows the BankHoliday infobox (1787ms)
    ✓ Shows the Transaction infobox (1743ms)


  4 passing (18s)
  1 failing

  1) Infobox results
       Shows the Taxon infobox:
     AssertionError: Timed out retrying after 10000ms: Expected to find content: 'Access to the countryside' within the selector: 'div.meta-results-panel h2' but never did.
      at Context.eval (webpack:///./cypress/e2e/infobox.cy.ts:17:7)




  (Results)

  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ Tests:        5                                                                                │
  │ Passing:      4                                                                                │
  │ Failing:      1                                                                                │
  │ Pending:      0                                                                                │
  │ Skipped:      0                                                                                │
  │ Screenshots:  1                                                                                │
  │ Video:        false                                                                            │
  │ Duration:     18 seconds                                                                       │
  │ Spec Ran:     infobox.cy.ts                                                                    │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘
```

--- a/src/backend/bigquery/bigquery.test.ts
+++ b/src/backend/bigquery/bigquery.test.ts
@@ -91,14 +91,14 @@ describe('[Function] sendInitQuery', () => {
     expect(BigQuery.prototype.query).toHaveBeenNthCalledWith(1, {
       query: `
         SELECT DISTINCT locale
-        FROM \`content.locale\`
+        FROM \`search.locale\`
         `,
       location: 'europe-west2',
       params: {},
     })
     expect(BigQuery.prototype.query).toHaveBeenNthCalledWith(2, {
       query: `
-        SELECT name
+        SELECT DISTINCT name
         FROM \`search.taxon\`
         `,
       location: 'europe-west2',
@@ -107,7 +107,7 @@ describe('[Function] sendInitQuery', () => {
     expect(BigQuery.prototype.query).toHaveBeenNthCalledWith(3, {
       query: `
         SELECT DISTINCT title
-        FROM \`graph.organisation\`
+        FROM \`search.organisation\`
         `,
       location: 'europe-west2',
       params: {},

--- a/src/backend/bigquery/bigquery.ts
+++ b/src/backend/bigquery/bigquery.ts
@@ -77,19 +77,19 @@ const sendInitQuery = async function (): Promise<InitResults> {
       await Promise.all([
         bigQuery(`
         SELECT DISTINCT locale
-        FROM \`content.locale\`
+        FROM \`search.locale\`
         `),
         bigQuery(`
-        SELECT name
+        SELECT DISTINCT name
         FROM \`search.taxon\`
         `),
         bigQuery(`
         SELECT DISTINCT title
-        FROM \`graph.organisation\`
+        FROM \`search.organisation\`
         `),
         bigQuery(`
         SELECT DISTINCT document_type
-        FROM \`content.document_type\`
+        FROM \`search.document_type\`
         `),
       ])
   } catch (error) {

